ext/openssl/openssl.stub.php

```diff
@@ -127,19 +161,24 @@ function openssl_get_privatekey($private_key, ?string $passphrase = null): OpenS
  */
 function openssl_pkey_get_details(OpenSSLAsymmetricKey $key): array|false {}
 
+/** @sensitive-param $password */
 function openssl_pbkdf2(string $password, string $salt, int $key_length, int $iterations, string $digest_algo = "sha1"): string|false {}
 
 function openssl_pkcs7_verify(string $input_filename, int $flags, ?string $signers_certificates_filename = null, array $ca_info = [], ?string $untrusted_certificates_filename = null, ?string $content = null, ?string $output_filename = null): bool|int {}
 
 /** @param OpenSSLCertificate|array|string $certificate */
 function openssl_pkcs7_encrypt(string $input_filename, string $output_filename, $certificate, ?array $headers, int $flags = 0, int $cipher_algo = OPENSSL_CIPHER_AES_128_CBC): bool {}
 
-/** @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key */
+/**
+ * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key
+ * @sensitive-param $private_key
+ */
 function openssl_pkcs7_sign(string $input_filename, string $output_filename, OpenSSLCertificate|string $certificate, $private_key, ?array $headers, int $flags = PKCS7_DETACHED, ?string $untrusted_certificates_filename = null): bool {}
 
 /**
  * @param OpenSSLCertificate|string $certificate
  * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string|null $private_key
+ * @sensitive-param $private_key
  */
 function openssl_pkcs7_decrypt(string $input_filename, string $output_filename, $certificate, $private_key = null): bool {}
```

Comment:
So this one is a bit tricky because you can pass private key string path / pem to the certificate parameter which is then taken as a private key so certificate might be sensitive as well. It's not great API though.

Reply:
Thanks for you taking a look at this. I can add the attribute to `$certificate` if you think that's useful. If it's only a technicality that is not likely to happen in practice we can also leave it off.

I would like to trust your recommendation with regard to developer experience vs security here.

Reply:
Okay, added.